### ISSUE TITLE
Spring tension to restore floating table

### ISSUE THREAD

**Issue by polemidis** [goscommons/CNC-Plasma-Table#15]
I created it in the prototype but have to make it in the CAD   ---Pictures needed---

instead of relying on the weight of the floating table+torch weight, a spring has to restore the floating table for reliability and stress free operation

**Comment by jurra** [comment]
This is also a simple task right? finding a proper spring that works for the weight of the torch head. @polemidis correct me if I am wrong.

**Comment by polemidis** [comment]
yeah pretty super easy. The weight is actually pulling to the same direction as the spring. But we cannot just count on the weight as dirt/friction etc my not be enough force to keep the torch head down. A small spring will do the job. I am satisfied with the rubber band it just look "cheap to the end user :)
